tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/missing_concept_record_suppression_test.py

```diff
@@ -0,0 +1,52 @@
+"""
```

Comment:
@jp3477 I don't think this unit test module is necessary.  It isn't covering anything that the integration test above isn't covering.  I recommend just removing it.

Reply:
Great, I was having trouble figuring out how to mock it up.